### DISCUSSION TITLE
mps-platform-docs/mps_internal/writing_mps_plugin_with_kotlin/

### DISCUSSION THREAD

**Discussion by giscus** [mbeddr/mps-platform-docs#9] [General]
# mps-platform-docs/mps_internal/writing_mps_plugin_with_kotlin/

This website collects practical knowledge about MPS and popular MPS platforms such as MPS-extensions and mbeddr. It is written mainly in the form of a Q&A.

http://mbeddr.com/mps-platform-docs/mps_internal/writing_mps_plugin_with_kotlin/

**Comment by nolisj** [comment]
Thanks! Bookmarking this for later use.
